ircbot.py

```diff
@@ -63,8 +63,21 @@ def _showError(self, failure):
     def command_live(self, rest):
         player = rest.partition(' ')
         if player:
-            # find the player on lichess
-            pass
+            try:
+                f = urllib.request.urlopen("http://en.lichess.org/api/user/" + username).read().decode("utf-8")
+            except urllib.error.HTTPError as err:
+                if(err.code == 404):
+                    return username + " was not found on Lichess.org"
+                return "HTTPError (" + str(err.code) + ") - " + err.reason
```

Comment:
Same as above, for string formatting stuff let's use `"{} blah blah {}".format(var1, var2)`